### ISSUE TITLE
Uncaught TypeError: tokens is not iterable

### ISSUE THREAD

**Issue by akonwi** [TypeStrong/atom-typescript#1505]
[Enter steps to reproduce:]

1. Open a typescript file
2. on some code, invoke the command `Typescript: Find References`

**Atom**: 1.40.0-nightly23 x64
**Electron**: 3.1.10
**OS**: Mac OS X 10.14.5
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package 13.2.1

### Stack Trace

Uncaught TypeError: tokens is not iterable

```
At /Users/akonwi/.atom/packages/atom-typescript/node_modules/atom-highlight/lib/highlight.js:81

TypeError: tokens is not iterable
    at highlightSync (/packages/atom-typescript/node_modules/atom-highlight/lib/highlight.js:81:41)
    at TsView.render (/packages/atom-typescript/dist/main/atom/components/tsView.js:15:22)
    at Object.initialize (/packages/atom-ide-datatip/node_modules/etch/lib/component-helpers.js:42:31)
    at /packages/atom-typescript/dist/main/atom/components/tsView.js:8:14)
    at render (/packages/atom-ide-datatip/node_modules/etch/lib/render.js:21:25)
    at addChildren (/packages/atom-ide-datatip/node_modules/etch/lib/render.js:45:24)
    at Object.render (/packages/atom-ide-datatip/node_modules/etch/lib/render.js:35:21)
    at Object.elementForItem (/packages/atom-typescript/dist/main/atom/views/simpleSelectionView.js:36:48)
    at listItems.items.map (/packages/git-plus/node_modules/atom-select-list/src/select-list-view.js:219:31)
    at Array.map (<anonymous>)
    at SelectListView.renderItems (/packages/git-plus/node_modules/atom-select-list/src/select-list-view.js:215:35)
    at SelectListView.render (/packages/git-plus/node_modules/atom-select-list/src/select-list-view.js:199:12)
    at updateSync (/packages/git-plus/node_modules/etch/lib/component-helpers.js:107:34)
    at /packages/git-plus/node_modules/etch/lib/component-helpers.js:72:7
    at DefaultScheduler.performUpdates (/packages/atom-ide-datatip/node_modules/etch/lib/default-scheduler.js:48:34)
```

### Commands

```
     -7:38.6.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -7:35.9.0 editor:consolidate-selections (input.hidden-input)
     -7:35.9.0 core:cancel (input.hidden-input)
     -7:35.2.0 fuzzy-finder:toggle-file-finder (input.hidden-input)
     -7:33.7.0 core:select-all (input.hidden-input)
     -7:32 core:confirm (input.hidden-input)
     -5:38.3.0 typescript:hide-signature-help (input.hidden-input)
     -5:38.3.0 vim-mode-plus:reset-normal-mode (input.hidden-input)
  6x -5:38 vim-mode-plus:move-up (input.hidden-input)
  2x -5:36.3.0 vim-mode-plus:move-to-previous-word (input.hidden-input)
     -5:35.3.0 command-palette:toggle (input.hidden-input)
     -5:34.2.0 core:confirm (input.hidden-input)
     -5:34.2.0 typescript:find-references (input.hidden-input)
     -5:32.8.0 editor:consolidate-selections (input.hidden-input)
     -5:32.8.0 core:cancel (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.8 
apm-publish 0.3.2 
atom-beautify 0.33.4 
atom-ide-datatip 0.11.0 
atom-ide-definitions 0.3.2 
atom-ide-diagnostics 0.0.0 
atom-ide-markdown-service 1.2.3 
atom-ide-signature-help 0.7.1 
atom-material-syntax-light 0.4.6 
atom-material-ui 2.1.3 
atom-typescript 13.2.1 
better-git-blame 0.4.7 
busy-signal 2.0.1 
city-lights-syntax 1.1.8 
city-lights-ui 1.5.3 
codestream 1.3.1 
codestream-atom-toolbar 0.0.1 
git-menu 3.0.1 
git-plus 8.7.1 
hyperclick 0.1.5 
ide-json 0.2.1 
ide-rust 0.21.0 
intentions 1.1.5 
lame-sauce-syntax-light 1.0.0 
language-babel 2.85.0 
language-docker 1.1.8 
language-kotlin 0.5.0 
language-rust 0.4.12 
linter 2.3.0 
linter-eslint 8.5.5 
linter-tslint 1.10.28 
linter-ui-plus 0.3.2 
notifications-plus 0.69.15 
one-black-syntax 1.14.0 
platformio-ide-terminal 2.9.1 
prettier-atom 0.56.6 
project-manager 3.3.7 
recent-files-fuzzy-finder 1.0.0 
relative-numbers 0.9.0 
split-diff 1.6.0 
sync-settings 0.8.6 
tool-bar 1.1.11 
vim-mode-plus 1.36.0 
```

**Comment by lierdakil** [comment]
Hello. Thanks for reporting! Sorry for the long silence, this got lost in the inbox.

I believe this has been fixed in v13.3.0, at least I can not reproduce with atom-typescript v13.3.0 and Atom v1.40.0-beta1. Please let me know if I am mistaken.

**Comment by akonwi** [comment]
It's resolved for me
